.storybook/gardenTheme.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { create } from '@storybook/theming/create';
+import { DEFAULT_THEME } from '../packages/theming/src';
+
+export const managerTheme = create({
+  base: 'light',
```

Comment:
I checked in with @allisonacs and we'd like to go with 'dark' mode (matching `react-containers`) in order to clearly differentiate between Storybook's chrome and Garden components.

Reply:
Updated to `dark` and have removed the `gridlines` and `background` essential plugins as we now have to override these values in our example previews.